extensions/amp-analytics/0.1/test/test-requests.js

```diff
@@ -58,6 +58,48 @@ describes.realWin('Requests', {amp: 1}, env => {
   }
 
   describe('RequestHandler', () => {
+    describe('send with request origin', () => {
```

Comment:
Can we have more test coverage on the 'origin' and 'baseUrl' behavior please. 
A few ideas
1. origin url is not absolute
2. both `origin` and `baseUrl` are absolute url
3. `baseUrl` doesn't have leading `/`, empty string, doesn't  exis

Please add more test cases you have in mind. Thank you